
float64 timestamp_seconds
---
UAVFPose pose